### PR TITLE
Interrupt on windows! 😯

### PR DESCRIPTION
**Pull request by fonsp**
YES!!!

It turns out that the magic trick was:
- https://learn.microsoft.com/en-us/windows/console/generateconsolectrlevent to send a Ctrl+C event to the process (this is not the same as a SIGINT signal on Windows). We can call this with `ccall`.
- To allow the process to handle the Ctrl+C event (and not its host, ie the Malt host), the process should be launched in [`detach`ed mode](https://docs.julialang.org/en/v1/base/base/#Base.Cmd). Distributed also does this.

### CODE REVIEW

File: src/Malt.jl [hunk 1]
```diff
@@ -581,18 +661,58 @@ Send an interrupt signal to the worker process. This will interrupt the
...
-        @warn "Malt.interrupt is not yet supported on Windows"
-        # _assert_is_running(w)
-        # _send_msg(w, MsgType.from_host_fake_interrupt, (), false)
-        nothing
+    if !isrunning(w)
+        @warn "Tried to interrupt a worker that has already stopped running." summary(w)
     else
-        Base.kill(w.proc, Base.SIGINT)
+        if Sys.iswindows()
+            ccall((:GenerateConsoleCtrlEvent,"Kernel32"), Bool, (UInt32, UInt32), UInt32(1), UInt32(getpid(w.proc)))
```
Comment (pankgeorg) [inline src/Malt.jl:668]: !!!

Comment (fonsp) [reply src/Malt.jl:668]: i did a thing 💖

Comment (fonsp) [reply src/Malt.jl:668]: and it's working!

Comment (fonsp) [reply src/Malt.jl:668]: proof: https://github.com/JuliaPluto/Malt.jl/actions/runs/6324554088/job/17174287331?pr=60#step:5:158

**Comment by fonsp** [comment]
It's working!! My new interrupt tests are only passing on Windows, because now Malt has better interrupt support on Windows than the other platforms 😅

**Comment by fonsp** [comment]
It's working! Just tweaking the tests now...

**Pull request merged**